CMakeLists.txt

```diff
@@ -28,8 +28,12 @@ endif()
 
 if(APPLE AND NOT CMAKE_OSX_DEPLOYMENT_TARGET)
     # If no deployment target has been set default to the minimum supported
-    # OS X version (this has to be set before the first project() call)
-    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.10 CACHE STRING "macOS Deployment Target")
+    # OS version (this has to be set before the first project() call)
+    if(CMAKE_SYSTEM_NAME STREQUAL "iOS")
+        set(CMAKE_OSX_DEPLOYMENT_TARGET 10.0 CACHE STRING "macOS Deployment Target")
+    else()
+        set(CMAKE_OSX_DEPLOYMENT_TARGET 10.10 CACHE STRING "macOS Deployment Target")
```

Comment:
Seeing this [change]( https://github.com/wxWidgets/wxWidgets/commit/4fb1764d6b1825af8e089b151650ceaa76e79666#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL32-R32) and also now knowing that the literal string for `set` is a docstring: Could the (second) instance of `macOS` be replaced with `iOS` here?

Reply:
Sure, it is just a description of the variable.